cmd/oci-runtime-tool/validate.go

```diff
@@ -465,17 +450,33 @@ func envValid(env string) bool {
 	return true
 }
 
-func capValid(capability string) bool {
-	for _, val := range defaultCaps {
-		if val == capability {
-			return true
-		}
-	}
+func capValid(c string, hostSpecific bool) bool {
+    cp := strings.ToUpper(c)
+
+    for _, cap := range capability.List() {
+        if cp == fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())) {
+            if hostSpecific && cap > lastCap() {
+				return false
+            }
+            return true
+        }
+    }
+
 	return false
 }
 
+func lastCap() capability.Cap {
```

Comment:
It looks like your new `capValid` code is also coming over [from generate][1].  Can we move all of that to a public validate function and import that into the generate library as well?

[1]: https://github.com/opencontainers/runtime-tools/blob/7ab77aa560f067c5a7cd4e67a37f6a73f117d07c/generate/generate.go#L930-L948